e2e/cypress/integration/system_console/reporting/site_statistics_spec.js

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('System Console > Site Statistics', () => {
+    it('MM-T904 Site Statistics displays expected content categories', () => {
```

Comment:
Thanks for covering tests both for enterprise and team edition. I've updated the TM4J test case. Please indicate test step,
```suggestion
    it('MM-T904_1 Site Statistics displays expected content categories', () => {
```